sdk/storage/Azure.Storage.Queues/src/Azure.Storage.Queues.csproj

```diff
@@ -29,6 +29,10 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="System.Memory.Data" />
+  </ItemGroup>
+
+  <!-- These packages are included as part net6.0; the external references are not needed. -->
+  <ItemGroup Condition="'$(TargetFramework)' != 'net6.0'">
```

Comment:
For maintainability should this be inverted so that adding future frameworks doesn't need the conditions to be touched?